### PR TITLE
Fix console expecter in tests

### PR DESCRIPTION
**Pull request by petrkotas**
**What this PR does / why we need it**:
Fixes the expecter behaviour. By adding the `\n` as the first command it makes sure that the expecter is receiving properly.

**Special notes for your reviewer**:

**Release note**:

```release-note
This patch fixes CI issues.
```

---
This change is [Reviewable](https://reviewable.io/reviews/kubevirt/kubevirt/1503)

### CODE REVIEW

File: tests/console_test.go [hunk 1]
```diff
@@ -38,25 +38,40 @@ var _ = Describe("Console", func() {
...
 		Expect(err).ToNot(HaveOccurred())
-		defer expecter.Close()
+		defer func() {
+			By("Closing the opened expecter")
+			expecter.Close()
+		}()
 
 		By("Checking that the console output equals to expected one")
 		_, err = expecter.ExpectBatch([]expect.Batcher{
+			&expect.BSnd{S: "\n"},
```
Comment (rmohr) [inline tests/console_test.go]: This one here really needs a `\n`, good catch.

File: tests/storage_test.go [hunk 2]
```diff
@@ -141,6 +141,7 @@ var _ = Describe("Storage", func() {
 
 				By("Checking that /dev/vdc has a capacity of 2Gi")
 				res, err := expecter.ExpectBatch([]expect.Batcher{
+					&expect.BSnd{S: "\n"},
```
Comment (rmohr) [inline tests/storage_test.go]: I think that the test here already use `tests.LoggedInAlpineExpecter(vmi)`, that should already take care of the `\n`.

Comment (rmohr) [reply tests/storage_test.go]: ok, actually the `\n` should also not be needed here, since we directly send something ...

**Comment by petrkotas** [comment]
@rmohr double checked, yesterday I shot with a shotgun full of `\n` to see how the CI reacts. Today I cleared it :). Running functests on my laptop now.

**Comment by petrkotas** [comment]
@rmohr if you agree at this phase, can we merge?

**Comment by vladikr** [comment]
@petrkotas can you please rebase? It has a conflict.

**Comment by petrkotas** [comment]
@vladikr @fabiand @rmohr did the rebase so now it is ready to be merged

**Comment by petrkotas** [comment]
@rmohr it stabilizes the tests with missing new line. Others are still timeouting randomly. I try to fix one thing at a time so the PRs are small and manageable.

**Comment by petrkotas** [comment]
@rmohr do you still require some changes? :)

File: tests/console_test.go [hunk 3]
```diff
@@ -38,25 +38,40 @@ var _ = Describe("Console", func() {
...
 	tests.PanicOnError(err)
 
 	BeforeEach(func() {
 		tests.BeforeTestCleanup()
 	})
 
-	RunVMIAndExpectConsoleOutput := func(vmi *v1.VirtualMachineInstance, expected string) {
+	JustBeforeEach(func() {
+		By("Opening new virtClient")
+		virtClient, err = kubecli.GetKubevirtClient()
```
Comment (rmohr) [inline tests/console_test.go]: Is this change also related to the expecter?

Comment (petrkotas) [reply tests/console_test.go]: @rmohr thanks, this change is related to the second PR I have in progress. This one is the concept I tested. Removing it now.

**Comment by petrkotas** [comment]
@rmohr ping?

**Comment by rmohr** [comment]
ph, @petrkotas you need to run `make` and commit the changes. Travis is failing.
